model/security/security.smithy

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@documentation("The message indicating the information of action of the model.")
+string Message
```

Comment:
I noticed that Message and MessageStatus are used in different security structures for output.So I decided to created this file to import this strings.